seaborn/matrix.py

```diff
@@ -298,7 +298,7 @@ def plot(self, ax, cax, kws):
 
         # setting vmin/vmax in addition to norm is deprecated
         # so avoid setting if norm is set
-        if "norm" not in kws:
+        if "norm" not in kws or not kws["norm"]:
```

Comment:
This depends on `Norm` objects having a positive "truthiness", which may work in practice but feels undefined from first principles. I suggest this more explicit check:

```suggestion
        if kws.get("norm") is not None:
```

Reply:
FYI, just double-checked your original comment now and I believe the equivalent to my statement would be `if kws.get("norm") is None`.